root/Makefile

```diff
@@ -92,7 +92,8 @@ release:: pre-release
 	@# Create a tag for our version
 	@git tag -d "$(APP_VERSION)" >&2 >/dev/null || true
 	@git tag "$(APP_VERSION)" >&2
-	@GORELEASER_CURRENT_TAG=$(APP_VERSION) goreleaser release --skip-announce --skip-publish --skip-validate --rm-dist
+	@GORELEASER_CURRENT_TAG=$(APP_VERSION) ASDF_GORELEASER_VERSION=$(shell cat "$(BOOTSTRAP_DIR)/.tool-versions" | grep goreleaser | awk '{ print $$2 }') \
```

Comment:
I accidentally snuck this in here, this fixes a small break introduced in #628. Given it's accidentally here... I'm leaning toward just going with it.

Reply:
I'm assuming this is here because of current working directory shenanigans?

Reply:
Correct!